extension/oidcauthextension/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/extension/oidcauthextension
+
+go 1.16
+
+require (
+	github.com/coreos/go-oidc v2.2.1+incompatible
+	github.com/stretchr/testify v1.7.0
+	go.opentelemetry.io/collector v0.33.1-0.20210820002854-d3000232f8f6
```

Comment:
Add a `go.opentelemetry.io/collector/model v0.33.1-0.20210820002854-d3000232f8f6  // indirect`

Reply:
```
	go.opentelemetry.io/collector/model v0.33.1-0.20210820002854-d3000232f8f6 // indirect
```

Reply:
Then run `go mod tidy` in this dir

Reply:
okay - will do